oci/client.py

```diff
@@ -453,6 +458,15 @@ def manifest(
 
         manifest_dict = res.json()
 
+        distribution_list_mediatype = 'application/vnd.docker.distribution.manifest.list.v2+json'
+        if (
+                'mediaType' in manifest_dict and
+                (mediaType := manifest_dict['mediaType']) == distribution_list_mediatype
+            ):
+            #XXX: ignore for now, need to be fixed later
+            raise NotImplementedError(f'OCI Manifest media type {mediaType}'
```

Comment:
nit: just pass the mediaType?
also: the comment "ignore for now" is a bit misleading. raising an exception is not quite the same as ignoring..

Reply:
mediaType: passing it directly would displease the linter and would destroy the fromatting.
Comment fixed.